.github/workflows/build_test_fmt.yml

```diff
@@ -7,22 +7,20 @@ on:
       master
 
 jobs:
-  build-and-test:
+  fmt-and-build:
     runs-on: ubuntu-latest
     env:
       RUSTFLAGS: "-D warnings"
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Check the coding style
-      run: |
-          cargo fmt -- --check
+      - uses: actions/checkout@v2
 
-    - name: Check for clippy warnings
-      run: cargo clippy
+      - name: Check the coding style
+        run: |
+          cargo fmt -- --check
 
-    - name: Build
-      run: cargo build
+      - name: Check for clippy warnings
+        run: cargo clippy
 
-    - name: Run Unit Tests and Documentation examples
-      run: cargo test
+      - name: Build
+        run: cargo build
```

Comment:
`cargo check` should be enough here.

Also, if you run this after `cargo clippy`, this is just redundant, since Clippy also runs all `rustc` checks. So I would either remove this step or run it as `cargo check` before Clippy, as an early exit. (Clippy is slightly slower than `cargo check`)

Reply:
Fixed in f252d6c